databases/mycli/Portfile

```diff
@@ -1,14 +1,12 @@
-# $Id$
-
 PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        dbcli mycli 1.5.2 v
+github.setup        dbcli mycli 1.8.1 v
 
-python.default_version     27
+python.versions     27 34 35
```

Comment:
Yeah I think I'm also not sure how it works. Given that I don't want to change it to `py-mycli` for now what should I do with `python.versions` and `python.default_version`?